iris_simple_example.ipynb

```diff
@@ -1,21 +1,76 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fca30380",
+   "metadata": {
+    "papermill": {
+     "duration": 0.004229,
+     "end_time": "2022-09-06T17:31:57.247207",
+     "exception": false,
+     "start_time": "2022-09-06T17:31:57.242978",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "# Find Label Errors Introduced in the Iris Classification Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91188b55",
+   "metadata": {
+    "papermill": {
+     "duration": 0.003233,
+     "end_time": "2022-09-06T17:31:57.253896",
+     "exception": false,
+     "start_time": "2022-09-06T17:31:57.250663",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "In this tutorial we show how `cleanlab` can be used to find label errors and improve the performance of a classifier model in classification tasks. We will use a modified version of the Iris dataset that has artificially introduced label errors to demonstrate this task.\n",
```

Comment:
Let's not call this tutorial, call it example instead. We want to clearly distinguish this repo from docs.cleanlab.ai